oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/oidc/web/OidcProviderConfigurationEndpointFilter.java

```diff
@@ -111,7 +111,8 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
 				.tokenIntrospectionEndpointAuthenticationMethods(clientAuthenticationMethods())
 				.subjectType("public")
 				.idTokenSigningAlgorithm(SignatureAlgorithm.RS256.getName())
-				.scope(OidcScopes.OPENID);
+				.scope(OidcScopes.OPENID)
+				.codeChallengeMethod("S256");
```

Comment:
@drunkcattt Can you please update the test in `OidcProviderConfigurationEndpointFilterTests` similar to `OAuth2AuthorizationServerMetadataEndpointFilterTests`.

Reply:
@drunkcattt Are you able to update the test in `OidcProviderConfigurationEndpointFilterTests`?

Reply:
It's been added.